package.json

```diff
@@ -11,7 +11,7 @@
     "lint": "eslint --ext .js,.vue ./",
     "test": "jest",
     "nuke": "rm -rf node_modules && yarn cache clean && yarn install",
-    "prepare": "husky install"
+    "prepare": "husky install && chmod +x .husky/*"
```

Comment:
this fixes an issue where the husky script(s) is not marked as executable, gotten from husky troubleshooting docs